examples/.vscode/settings.json

```diff
@@ -1,5 +1,6 @@
 {
     // Use a custom PowerShell Script Analyzer settings file for this workspace.
     // Relative paths for this setting are always relative to the workspace root dir.
-    "powershell.scriptAnalysis.settingsPath": "./PSScriptAnalyzerSettings.psd1"
+    "powershell.scriptAnalysis.settingsPath": "./PSScriptAnalyzerSettings.psd1",
+    "files.defaultLanguage": "powershell"
```

Comment:
Since vscode-powershell is typescript project, should defaultLanguage be `typescript`?